### ISSUE TITLE
[Bug]: Storage Cracker level has both test cases with the passcode as '0'

### ISSUE THREAD

**Issue by OrangeOliviero** [Turing-Complete-Game/Suggestions-and-Issues#179]
### What happened?

As the subject said - there are just two test cases for this level, both had the passcode be 0.

### Version

Latest

### Operating System

Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MegaIng** [comment]
This is not actually the case, it's a broken adversarial checker. This is a known issue.

**Comment by OrangeOliviero** [comment]
I mean, both cases only accepted when I output 0 and gave me a response of "too high" with any other number, whatever the cause.

Just being clear in case these do wind up being two separate issues.

**Comment by MegaIng** [comment]
For any other number? E.g. you tried randomly choosing other numbers, not just going down the powers of 2?

**Comment by OrangeOliviero** [comment]
My program started with an initial guess of 128 and a step of 64, and then adds/subtracts the step based on whether the response was too high or not, then halves the step and repeats.

So it would guess 128 - input would be "too high" (1), so then:
- guess 64 - too high
- guess 32 - too high
- guess 16 - too high
- guess 8 - too high
- guess 4 - too high
- guess 2 - too high
- guess 1 - too high
- guess 0 - accepted

**Comment by MegaIng** [comment]
Yes, this is exactly going down the powers of two. If you chose any other numbers it wouldn't work (or if you weren't reading the input)

**Comment by GufNZ** [comment]
Close?
